/* 
 * File:   LogitechCamera.hpp
 * Author: satya
 *
 * Created on December 12, 2013, 8:49 PM
 */

#ifndef LOGITECHCAMERA_HPP
#define	LOGITECHCAMERA_HPP

class LogitechCamera : Sensor {
public:
    LogitechCamera();
    LogitechCamera(const LogitechCamera& orig);
    virtual ~LogitechCamera();
private:

};

#endif	/* LOGITECHCAMERA_HPP */

